share/spack/gitlab/cloud_pipelines/stacks/developer-tools-manylinux2014/spack.yaml

```diff
@@ -0,0 +1,89 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=x86_64_v3
+  concretizer:
+    unify: true
+  definitions:
+  - default_specs:
+      # editors
```

Comment:
Does it make sense to include clang and gnu compilers here? Or do you include these in the build cache create by building the image?

Reply:
Both gcc and clang are built as part of building this stack, but the gcc is pulled in by emacs. The clang is explicitly built by the llvm item. Neither are really meant to be used _as compilers_ where I use them currently, they're there to provide clang-tools* and libgccjit and similar tooling.  There's no reason they can't be, but they wont necessarily be base-system compatible if they come out of a bin-cache.